unittests/operations_tests/router_algorithms_tests/test_routers.py

```diff
@@ -448,6 +450,8 @@ def test_simple(params):
 def test_self(params):
     algorithm, _n_vertices, n_m_vertices = params
     unittest_setup()
+    # TODO check after
+    #  https://github.com/SpiNNakerManchester/PACMAN/pull/555
```

Comment:
This has been merged - should something be changed here?

Reply:
That pr did not take into account the way spin2 splits the links and processors.

prefer to do that in a seperate followup pr

Reply:
To be done later